README.md

```diff
@@ -52,13 +52,13 @@ necessarily send the first packet, this distinction is just made to make the all
 
 ### Opening a new stream
```

Comment:
In the section above ^^, we should also get rid of all information about stream IDs, initiators, and receivers.

Reply:
> we should also get rid of all information about stream IDs, initiators, and receivers.

There is one part where mentioning initiators makes sense - when talking about flag's oddness.